ext/pdo_sqlite/tests/gh7803.phpt

```diff
@@ -0,0 +1,36 @@
+--TEST--
+GH-7803: PDO __call breaks method calls
+--FILE--
```

Comment:
We need to explicitly declare the required extensions (pdo_sqlite is not loaded on AppVeyor by default):
```suggestion
--EXTENSIONS--
pdo_sqlite
--FILE--
```